tools/breaking-changes/data/breaking-changes.json

```diff
@@ -26453,7 +26453,7 @@
         "apiElementKind": "Class",
         "entryPoint": "@spartacus/checkout/root",
         "deletedComment": "// TODO:Spartacus - Method 'paymentProcessSuccess' was removed from Class 'CheckoutPaymentFacade'.",
-        "migrationComment": "",
+        "migrationComment": "After switching to commands & queries, processes are no longer used.",
```

Comment:
Looking at 4.3.x, it looks like the 'loading state' for the process when setting a payment method. So in this case, it would be leveraging setPaymentDetails from CheckoutPaymentFacade